pkg/inventory/deployment.go

```diff
@@ -28,6 +29,7 @@ func ForDeployments(existing []appsv1.Deployment, desired []appsv1.Deployment) D
 
 			// we can't blindly DeepCopyInto, so, we select what we bring from the new to the old object
 			tp.Spec = v.Spec
+			tp.Spec = inject.PropagateOAuthCookieSecret(&t.Spec, &tp.Spec)
```

Comment:
You can have both lines 31 and 32 done on the same line:

```
tp.Spec = inject.PropagateOAuthCookieSecret(&t.Spec, v.Spec)
```

You'll need to change your arguments to accept values instead of pointers, which is in line with what we discussed in a previous review :)